src/mcts/utils.h

```diff
@@ -151,6 +154,34 @@ class MctsStats {
   // std::mutex mSelf_;
 };
 
+// ChatGPT
```

Comment:
What's this entirely commented function? Why is it here?

Reply:
I asked ChatGPT to improve the code. I didn't end up using it, but thought we might want to consider adopting it in the future. Thoughts?

Reply:
I think that whoever is reading the code will not know that this is a "better version" of the function below. Indeed that's not guaranteed and in fact unlikely, because this version isn't tested. Surely there's a bug or two.

You should either delete this (recommended) or test it and uncomment it and delete the old version (not recommended, work).

Reply:
I'll just delete it, thanks